2-mandatory/4-people-i-know.js

```diff
@@ -403,15 +415,29 @@ This time, I only want the full names of the people who can multitask
 
 let colleaguesWhoCanMultitask = [];
 
+friends
+  .map((friend) => friend.colleagues)
+  .forEach((obj) => {
+    for (const key in obj) {
+      if (obj[key].skills.includes("Multi-tasking")) {
+        colleaguesWhoCanMultitask.push(obj[key].name);
+      }
+    }
+  });
+
```

Comment:
Good use of .map and .forEach methods. I have used the same methods but in reverse order, that has saved me from using the for...in statement. Both work great.